precommend/.pre-commit-config.yaml

```diff
@@ -60,6 +60,7 @@ repos:
     rev: v18.1.1
     hooks:
       - id: clang-format  # Format C++ code with Clang-Format - automatically applying the changes
+        types_or: [c++, c, cuda]
```

Comment:
I am including `objective-c` in https://github.com/ssciwr/precommend/blob/main/precommend/rules.py#L66, so it should also be here.

Reply:
Oh and same holds for `c#`.

Reply:
good point! fixed